Sources/OVP/Parsers/OVPObjectMapper.swift

```diff
@@ -29,7 +29,7 @@ class OVPObjectMapper: NSObject {
             switch name {
             case "KalturaMediaEntry":
                 return OVPEntry.self
-            case "KalturaLiveStreamEntry":
+            case "KalturaLiveStreamEntry", "KalturaLiveStreamAdminEntry":
```

Comment:
Need to see where this object type came from and if it's supposed to be parsed.
Checking...

Reply:
This is a legacy object done for QA.